CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.5.4dev
 
+* [Fix] Doc Typo `PosgreSQL` in Connecting
```

Comment:
```suggestion
```

as a rule, we only list things in the changelog if they impact end users. so, only big changes in the docs (new sections, revamped sections) should go in the changelog

Reply:
Gotcha, we only add to changelog when the package itself it gets affected, we can exclude the DOC update part.

Fix in [ed67f91](https://github.com/ploomber/jupysql/pull/110/commits/ed67f91c8a98f9d2a96ac61e0e18fdc7fdf50610)